Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp

```diff
@@ -19994,6 +19995,11 @@ IGNORE_CLANG_WARNINGS_END
         std::sort(cases.begin(), cases.end());
         switchStringRecurse(data, buffer, length, cases, 0, 0, cases.size(), 0, false);
 
+        m_out.appendTo(isRopeBlock, slowBlock);
+        const UnlinkedStringJumpTable& unlinkedTable = m_graph.unlinkedStringSwitchJumpTable(data->switchTableIndex);
+        LValue ropeLength = m_out.load32NonNegative(string, m_heaps.JSRopeString_length);
+        m_out.branch(m_out.belowOrEqual(m_out.sub(ropeLength, m_out.constInt32(unlinkedTable.minLength())), m_out.constInt32(unlinkedTable.maxLength() - unlinkedTable.minLength())), unsure(slowBlock), unsure(lowBlock(data->fallThrough.block)));
+
```

Comment:
Ditto here: Slow case falls through to the Rope case.

Reply:
Ditto. switchStringRecurse always ends with jump to `data->fallThrough.block`'s fallthrough.